content/projects/gsoc/2021/projects/cloudevents-plugin.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: gsocproject
+title: "CloudEvents plugin for Jenkins"
+goal: "Be able to listen to and emit cloudevents from Jenkins."
+category: Plugins
+year: 2021
+sig: cloud-native
+status: "Active"
+student: adi10hero
+mentors:
+- "waveywaves"
+- "salaboy"
+- "jeffpearce"
+tags:
+- gsoc2021
+- cloudevents
+- cloudnative
+links:
+  draft: TODO
+  chat: TODO
```

Comment:
In talks w/ one of the mentors of this project to start a chat-room in Jenkins' Gitter.

Reply:
chat:
Gitter: https://gitter.im/jenkinsci/cloudevents-plugin
Slack: https://app.slack.com/client/TJWP1JXK6/C022S35AANP